<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>toOracleSettingUI</class>
  <widget class="QWidget" name="toOracleSettingUI" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>361</width>
        <height>287</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Form1</string>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item row="1" column="0" >
        <widget class="QLabel" name="TextLabel6_2_2_2" >
          <property name="text" >
            <string>&amp;Default date format</string>
          </property>
          <property name="buddy" >
            <cstring>DefaultDate</cstring>
          </property>
          <property name="toolTip" >
            <string>The default dateformat to use when querying the database.</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="0" column="0" >
        <widget class="QLabel" name="TextLabel6_2_3" >
          <property name="text" >
            <string>&amp;Checkpoint name</string>
          </property>
          <property name="buddy" >
            <cstring>CheckPoint</cstring>
          </property>
          <property name="toolTip" >
            <string>The name of the checkpoint used when needed to rollback work.</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="1" column="1" colspan="2" >
        <widget class="QLineEdit" name="DefaultDate" />
      </item>
      <item rowspan="1" row="0" column="1" colspan="2" >
        <widget class="QLineEdit" name="CheckPoint" />
      </item>
      <item row="3" column="0" >
        <widget class="QLabel" name="TextLabel6_3_2" >
          <property name="text" >
            <string>&amp;Max long and LOB data length</string>
          </property>
          <property name="buddy" >
            <cstring>ExplainPlan</cstring>
          </property>
          <property name="toolTip" >
            <string>Max length to read from LOB or LONG data fields. LONG:s can not be specified unlimited but will be defaulted to 33000 if unlimited.</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="3" column="2" >
        <widget class="QCheckBox" name="Unlimited" >
          <property name="text" >
            <string>Unlimited</string>
          </property>
          <property name="checked" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="3" column="1" >
        <widget class="QLineEdit" name="MaxLong" >
          <property name="enabled" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="6" column="2" >
        <spacer name="Spacer6" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item rowspan="1" row="5" column="0" colspan="1" >
        <spacer name="Spacer2_2" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="5" column="1" >
        <widget class="QPushButton" name="CreatePlanTable" >
          <property name="enabled" >
            <bool>false</bool>
          </property>
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
            </sizepolicy>
          </property>
          <property name="text" >
            <string>Create Table</string>
          </property>
        </widget>
      </item>
      <item row="5" column="2" >
        <widget class="QCheckBox" name="VsqlPlans" >
          <property name="text" >
            <string>&amp;V$SQL_PLAN</string>
          </property>
          <property name="toolTip" >
            <string>Use V$SQL_PLAN when possible.</string>
          </property>
        </widget>
      </item>
      <item row="4" column="0" >
        <widget class="QLabel" name="TextLabel6_3" >
          <property name="text" >
            <string>&amp;Explain plan table</string>
          </property>
          <property name="buddy" >
            <cstring>ExplainPlan</cstring>
          </property>
          <property name="toolTip" >
            <string>Name of the table to put explain plan output in.</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="4" column="1" >
        <widget class="QLineEdit" name="ExplainPlan" />
      </item>
      <item row="4" column="2" >
        <widget class="QCheckBox" name="KeepPlans" >
          <property name="text" >
            <string>&amp;Keep plans</string>
          </property>
          <property name="toolTip" >
            <string>Keep records for the plan explanations in the plan table.</string>
          </property>
        </widget>
      </item>
      <item row="2" column="2" >
        <widget class="QSpinBox" name="OpenCursors" >
          <property name="minimum" >
            <number>1</number>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="2" column="0" colspan="2" >
        <widget class="QLabel" name="TextLabel1" >
          <property name="text" >
            <string>Maximum cursors to keep open</string>
          </property>
          <property name="buddy" >
            <cstring>OpenCursors</cstring>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <tabstops>
    <tabstop>CheckPoint</tabstop>
    <tabstop>DefaultDate</tabstop>
    <tabstop>OpenCursors</tabstop>
    <tabstop>MaxLong</tabstop>
    <tabstop>Unlimited</tabstop>
    <tabstop>ExplainPlan</tabstop>
    <tabstop>KeepPlans</tabstop>
    <tabstop>CreatePlanTable</tabstop>
    <tabstop>VsqlPlans</tabstop>
  </tabstops>
  <connections>
    <connection>
      <sender>CreatePlanTable</sender>
      <signal>clicked()</signal>
      <receiver>toOracleSettingUI</receiver>
      <slot>createPlanTable()</slot>
    </connection>
    <connection>
      <sender>Unlimited</sender>
      <signal>toggled(bool)</signal>
      <receiver>MaxLong</receiver>
      <slot>setDisabled(bool)</slot>
    </connection>
  </connections>
</ui>
